tests/test_operation.py

```diff
@@ -134,6 +134,9 @@ def test_operation_init(self, test_class, monkeypatch):
         if test_class in (qml.ControlledQubitUnitary, qml.MultiControlledX):
             pytest.skip("ControlledQubitUnitary alters the input params and wires in its __init__")
 
+        if test_class == qml.PauliError:
```

Comment:
How come this check was required?

Reply:
because PauliError has a unique signature